aip/client-libraries/4223.md

```diff
@@ -19,7 +19,7 @@ specific request fields.
 
 Client libraries **must not** implement client-side payload validation based
 on the `google.api.field_behavior` annotation, except to prevent
-[local failures](4223.md#local-failures).
+[local failures](4223#local-failures).
```

Comment:
Did you double check that this actually works? I think if it's an anchor in the same API we don't need the number, just the anchor...but if this works, great!